### ISSUE TITLE
Create New Employee (Admin Mode) - Toasts, notifications and status' should be inside an aria-live-region

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#265]
<img width="842" alt="Screen Shot 2020-04-27 at 12 29 22 PM" src="https://user-images.githubusercontent.com/31868510/80506087-21bbee80-8943-11ea-87f9-eea12ae17930.png">

Dynamic content which updates without a page reload is generally either a region or a widget. Simple content changes which are not interactive should be marked as live regions. Live regions will alert screen readers and other assistive technologies to changes in page content.

You can read more about ARIA live regions here: 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions
